lib/src/migrator.dart

```diff
@@ -74,8 +75,11 @@ abstract class Migrator extends Command<Map<Uri, String>> {
   Map<Uri, String> run() {
     var allMigrated = <Uri, String>{};
     var importer = FilesystemImporter('.');
```

Comment:
Is there a reason why this is not also a `ReversibleFilesystemImporter`?

Reply:
URLs that are relative to the entrypoint should always be converted to relative URLs. In fact, generating a `@use` whose URL is relative to the working directory would be incorrect, since the working directory is not on the load path.